src/index.html

```diff
@@ -1,13 +1,309 @@
+          Browse Nothing
+          <br>
+          products by category
+        </h2>
+
+        <section class="category-block">
+          <a href="#" class="category-block__item--1">
+            <img
+              class="category-block__image"
+              src="images/parrots-phone.png"
+              alt="parrots phone"
+            >
+          </a>
+
+          <a href="#" class="category-block__item--2">
+            <img
+              class="category-block__image"
+              src="images/headphones-hand.png"
+              alt="headphones hand"
+            >
+          </a>
+        </section>
+
+        <h3 class="name category-block__name">All products</h3>
+
+
+        <section class="category-block">
+          <a href="#" class="category-block__item--2">
+            <img
+              class="category-block__image"
+              src="images/headset.png"
+              alt="headset"
+            >
+          </a>
+
+          <a href="#" class="category-block__item--1">
+            <img
+              class="category-block__image"
+              src="images/ear-stick-hand.png"
+              alt="ear stick hand"
+            >
+          </a>
+        </section>
+        <h3 class="name category-block__name">Audio</h3>
+
+        <section class="category-block">
+          <a href="#" class="category-block__item--1">
+            <img
+              class="category-block__image"
+              src="images/phone.png"
+              alt="phone"
+            >
+          </a>
+
+          <a href="#" class="category-block__item--2">
+            <img
+              class="category-block__image"
+              src="images/charger.png"
+              alt="charger"
+            >
+          </a>
+        </section>
+        <h3 class="name category-block__name">Accessories</h3>
+
+        <button
+          type="submit"
+          class="btn category-block__btn">
+          View all
+        </button>
+      </section>
+
+
+
+      <section class="about-us" id="login">
+
+        <div class="about-us__conteiner">
+
+          <h2 class="title about-us__title">About us</h2>
+
+          <p class="description about-us__description">
+            We&#96;re a London-based tech company on a mission to remove barriers between people and technology.
+            <br> <br>
+            To do this, we&#96;re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+            No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we&#96;re proud to share with our friends and family. Simple.
+            <br> <br>
+            Because beautiful tech shouldn&#96;t be complicated.
+          </p>
+        </div>
+      </section>
+
+
+      
+      <section class="contact-us" id="cart">
+        <h2 class="title contact-us__title">Contact us</h2>
+
+        <form
+        class="feedback-form"
+        method="get"
+        autocomplete="on"
+        >
```

Comment:
```suggestion
        <form
          class="feedback-form"
          method="get"
          autocomplete="on"
        >
```
pay attention to formatting
also check it 15 [here](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md) - might be useful if you will use `button` for Submit